doc/nrf/ug_bootloader.rst

```diff
@@ -7,212 +7,185 @@ Secure bootloader chain
    :local:
    :depth: 2
 
-The |NCS| provides a secure bootloader solution based on the chain of trust concept.
+The core architecture behind the secure bootloader solution provided by |NCS| is based on the *chain of trust* concept.
 
-By using this bootloader chain, you can easily ensure that all code that is executed is authorized and your application is protected against running altered code.
-If, for example, an attacker attempts to modify your application, or you have a bug in your code that overwrites parts of the application flash, the secure bootloader chain detects that the flash has been altered and your application does not start up.
+By using this secure bootloader chain, you can easily ensure that all code that is executed is authorized and your application is protected against running altered code.
+If, for example, an attacker attempts to modify your application, or you have a bug in your code that overwrites parts of the application firmware image, the secure bootloader chain detects that the firmware has been altered and your application does not start up.
+
+You should choose the structure of the secure bootloader chain depending on your desired firmware update scheme:
```

Comment:
```suggestion
When choosing the structure of the secure bootloader chain, make the decision based on your desired firmware update scheme:
```
`should` sounds like `you can, but you don't have to`, and I don't know if this is what is intended here.

Reply:
I mean whether need to update application  or app and mcuboot.

Reply:
Ah yes. This is mentioned in the bullets following this statement.

Reply:
Ah yes, this is brought up in the bullets following this statement.